commodore/postprocess/builtin_filters.py

```diff
@@ -31,7 +32,9 @@ def _builtin_filter_helm_namespace(
         create_namespace = "true" if create_namespace else "false"
     exclude_objects = kwargs.get("exclude_objects", [])
     exclude_objects = "|".join([json.dumps(e) for e in exclude_objects])
-    output_dir = _output_dir(work_dir, component.name, path)
+    # NOTE: we pass "" for compiled_dir here, as we already patch `path` to
```

Comment:
Sounds like a redundancy that could be eliminated.

Reply:
yeah, I looked at it again, and figured out that I implemented a workaround for a bug in the postprocessing in this PR :see_no_evil: 

See #357 for the correct fix.